File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main (void)
{
    //Example of a constant: const int n = 3;
    //Get size of grid
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1);

    //Print grid of Bricks
    for (int i = 0; i < n; i++)
    {
         for (int j = 0; j < n; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/lecture6/mario.py
# Prints a column of bricks, using a helper function to get input

from cs50 import get_int


def main():
    height = get_height()
    for i in range(height):
        print("#")


def get_height():
    while True:
        # Notice that try is utilized to attempt to convert n to an integer.
        # If it cannot do so, an error is outputted.
        try:
            n = int(input("Height: "))
            if n > 0:
                return n
        except ValueError:
            print("Not an integer")


main()